app/views/user_mailer/send_password.html.erb

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
```

Comment:
Not sure you will need the main tags as you are rendering them from layouts/mailer.html.erb.